OracleTuxedo/core/samples/shm/build.sh

```diff
@@ -2,4 +2,5 @@
 docker build -t oracle/tuxedoshm .
 
 echo "To run the sample, use:"
-echo "docker run -ti -v \${Local_volumes_dir}/TuxedoVolumes/\${VERSION}:/u01/oracle/user_projects oracle/tuxedoshm /bin/bash"
+echo "docker run -d --name tuxedoshm -v \${Local_volumes_dir}/TuxedoVolumes/\${VERSION}:/u01/oracle/user_projects oracle/tuxedoshm /bin/bash"
+echo "docker start tuxedoshm; docker attach tuxedoshm"
```

Comment:
The `docker run` command will automatically start the container. And you shouldn't need to attach to a running container at all. This line should be removed.

Rather, you probably should provide instructions on how to test/validate the application is working from outside the containers.